Packs/AHA/Integrations/AHA/README.md

```diff
@@ -0,0 +1,84 @@
+Use the Aha! integration to list and manage Cortex XSOAR features from Aha.
+This integration was integrated and tested with API version 1.0 of Aha
+
+## Configure Aha on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Aha.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Server URL |  | True |
+    | Api Key | API Key to access service REST API  | True |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### aha-get-features
+***
+Will list all features from service, unless a specific feature is specified
+
+
+#### Base Command
+
+`aha-get-features`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| from_date | Show features created after that date. Default is 2020-01-01. | Optional | 
+| feature_name | get a specific feature. Possible values are: . | Optional | 
```

Comment:
`possible values are` ? :)

Reply:
not sure about this. i have given possible values in command example section.